src/components/ScaffoldVuer.vue

```diff
@@ -2095,6 +2447,20 @@ export default {
   &:hover {
     cursor: pointer;
   }
+
+  &:focus {
+    border: none;
+    outline: none;
+  }
+
+  &.shape {
+    margin-left: 4px;
+    margin-right: 4px;
+    color: var(--el-color-primary-light-5);
```

Comment:
Two color here need to add `!important`. Otherwise, the tool icon style will be overwritten by mapvuer.

Reply:
Thanks for spotting this.